include/oneapi/dpl/internal/dynamic_selection_impl/submission_group.h

```diff
@@ -0,0 +1,50 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_SUBMISSION_GROUP_H
+#define _ONEDPL_SUBMISSION_GROUP_H
+
+#include "oneapi/dpl/internal/dynamic_selection_impl/concurrent_queue.h"
+#include <vector>
+
+namespace oneapi {
+namespace dpl {
+namespace experimental {
+  template<typename T>
+  class submission_group_t {
```

Comment:
I do not see this class (actually, the whole header) being used anywhere, not even in tests. Why is it needed?

Reply:
Header removed.